README.md

```diff
@@ -14,7 +14,14 @@ Running pre-commit will start docker-compose with Postgres and our code then run
 
 The database scripts live in [verify-event-system-database-scripts](https://github.com/alphagov/verify-event-system-database-scripts).
 
-The tests currently rely on scripts in the migrations directory of the above repo.
+The `./build/run-tests.sh` script expects to find the above repo, checked out, in the `../verify-event-system-database-scripts`
+directory. If it does not find it there, it will automatically clone it to that location.
```

Comment:
This feels a bit weird. I still kinda wonder whether the schema and the things that effectively depend on it should be in a mono-repo.

Reply:
We moved to its own repo last quarter, the schema used to be in this repo.

I agree it's not ideal, but there is precedent for this with the hub tests relying on different repos being in specific locations.

We have two repos that are reliant on the schema migrations (the reporter and the recorder), and we'll need to make similar changes to the reporter repo, as those tests will have similar issues now too.